examples/multi_label/classifier_chain.py

```diff
@@ -0,0 +1,86 @@
+"""
+============================
+Classifier Chain
+============================
+
+Demonstrate that a single classifier chain
+out performs 10 independent classifiers
+and that an ensemble of 10 randomly ordered
+classifier chains out performs a single classifier chain
+
+"""
+
+import numpy as np
+from sklearn.multi_label import ClassifierChain
+from sklearn.model_selection import train_test_split
+from sklearn.multiclass import OneVsRestClassifier
+from sklearn.metrics import jaccard_similarity_score
+from sklearn.linear_model import LogisticRegression
+from sklearn.datasets import fetch_rcv1
+
+# Get the Reuters Corpus Volume I dataset
+rcv1 = fetch_rcv1()
```

Comment:
I was actually not talking about the download (though that's also an issue) but about the computation. Have you tried yeast? That should have some correlations. And if ``make_multilabel_classification`` is not helpful for multi-label classification we should change/fix it.